doc/performance-investigations/0015-feedback-journey-performance-test/summary.md

```diff
@@ -0,0 +1,53 @@
+# Feedback performance investigation
+
+This test is to investigate the performance of feedback journey. Limit of feedback submissions was extended to 5 to allow a single journey to exercise more than 2 feedback (to facilitate more requests with fewer launches/submissions). `census_household_gb_eng.json` journey was used as a baseline. New requests file was generated from `test_feedback.json` schema for this investigation.
+
+## Benchmark version
+
+| Tag | Commit |
+|--------|-------|
+| feedback-performance-investigation  | 38ae1812858d9b1a4d204fc336adad6a63831f1c
+
+## Benchmark profile
+
+| Option | Value |
+|--------|-------|
+| Requests file | census_household_gb_eng.json |
+| Run time | 20 min |
+| User wait time minimum | 1 |
+| User wait time maximum | 2 |
+| Clients | 144 |
+| Hatch rate | 144 |
+| Locust Instances | 1 |
+
+## Results
+
+Results based on 99th percentile timings.
+
+| Metric | Baseline | Investigation |
+|--------|----------|---------------|
+| Questionnaire GETs | 118 | 74 |
+| Questionnaire POSTs | 141 | 121 |
+| All requests | 129 | 86 |
+| Total requests (over 20 mins) | 1,924,318 | 1,484,407 |
```

Comment:
I don't quite understand why the investigation is almost 500k fewer requests if they were run for the same duration?. Are we loosing out that much purely because of the greater number of launches and submission in the investigation?

Reply:
We do submit five times every launch, I was asked to extend the limit of feedback submissions over 2, so did 5.